query-service-impl/src/test/resources/application.conf

```diff
@@ -39,6 +39,18 @@ service.config = {
         startTimeAttributeName = "Trace.start_time_millis"
         slowQueryThresholdMs = 0
         percentileAggFunction = "PERCENTILE"
+        additionalTenantFilters = [
```

Comment:
There are multiple problems in the way this new config has been defined:
- It is  is very confusing. Only after going through the code one can understand.
- There is a hardcoded assumption that `startTimeAttributeName` will act as the lhs key for the 2 values below. Makes it very specific and no room for generalising.
- It is hardcoded with `OR` operator.

I would prefer it being defined as follows:
```
additionalHandlerConfigs = [
  operator = "OR"
  {
    key = "Trace.start_time_millis"
    operator = LT
    value = "max_cutoff"
  },
  {
    key = "Trace.start_time_millis"
    operator = GE
    value = "min_cutoff"
  },
  filter : {
    columnIdentifiier: ...
  }
]
```

Reply:
Would `operator = "AND"` possible here?

Reply:
We would like to apply the time range to `startTimeAttributeName`. Any other attribute can go to the normal filter expression.

The given time range indicates that we would like to apply a filter within that specified range. is that not clear?